servlet.py

```diff
@@ -251,17 +256,15 @@ def get(self):
         query = self.get_argument('q', default='pairs')
 
         if query == 'pairs':
-            responseData = []
-            for pair in self.pairs:
-                (l1, l2) = pair.split('-')
-                responseData.append({'sourceLanguage': l1, 'targetLanguage': l2})
-                if self.get_arguments('include_deprecated_codes'):
-                    responseData.append({'sourceLanguage': toAlpha2Code(l1), 'targetLanguage': toAlpha2Code(l2)})
-            self.sendResponse({'responseData': responseData, 'responseDetails': None, 'responseStatus': 200})
-        elif query == 'chains':
-            src = self.get_argument('src', default=None)
+            src = self.get_argument('chainSource', default=None)
             if not src:
-                self.send_error(400, explanation='Missing argument src')
+                responseData = []
+                for pair in self.pairs:
+                    (l1, l2) = pair.split('-')
+                    responseData.append({'sourceLanguage': l1, 'targetLanguage': l2})
+                    if self.get_arguments('include_deprecated_codes'):
+                        responseData.append({'sourceLanguage': toAlpha2Code(l1), 'targetLanguage': toAlpha2Code(l2)})
+                self.sendResponse({'responseData': responseData, 'responseDetails': None, 'responseStatus': 200})
             else:
                 self.sendResponse({
                     'responseData': list(self.paths[src].keys()),
```

Comment:
Can we actually send the response in the same format? I know it's redundant but then the API documentation is simpler and shorter. Client code also has fewer branches.